src/scripts/main.js

```diff
@@ -1 +1,70 @@
 'use strict';
+
+let click = false;
+let contextmenu = false;
+
+function getNotification(className, message) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = className;
+  notification.textContent = message;
+  document.body.append(notification);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  addEventListener('click', () => {
```

Comment:
Attaching event listeners globally without removing them can lead to memory leaks. Consider removing the event listeners once they are no longer needed.